application/templates/partials/whatsNew.njk

```diff
@@ -2,9 +2,9 @@
   <h3 id="whats-new" class="govuk-heading-m">What’s new</h3>
 
   <ul class="govuk-list govuk-list--bullet">
+  <li>23 September 2024: Account header <a href="/hmrc-design-patterns/hmrc-design-patterns-archive/" class="govuk-link"> archived</a> </li>
```

Comment:
Small change but the link shouldn't have a space in it:
```suggestion
  <li>23 September 2024: Account header <a href="/hmrc-design-patterns/hmrc-design-patterns-archive/" class="govuk-link">archived</a> </li>
```

Reply:
Thanks Tim, I'll change this